src/core/jquery-enhanced.js

```diff
@@ -50,57 +50,7 @@ window.$.fn.renameElement = function(name) {
 // This method will publish a warning if a title is used on a definition
 // instead of an @lt (as per specprod mailing list discussion).
 window.$.fn.getDfnTitles = function(args) {
```

Comment:
@sidvishnoi @marcoscaceres Why do we have to keep this extension? Is there any compat issue?

Reply:
Removing this will require fixing tests, though.

Reply:
I think we should mark them as deprecated, as it's easy enough in this case.

Reply:
Absolutely. We should console.warn and start seeing if there will be any damage in yanking support.